### PR TITLE
RsPaymentTerminal: initial commit

### PR DESCRIPTION
**Pull request by dorezyuk**
Initial commit for the payment terminal

### CODE REVIEW

File: modules/Cargo.toml [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
    "RsIskraMeter",
+   "RsPaymentTerminal",
    "rust_examples/RsExample",
    "rust_examples/RsExampleUser",
 ]
 
 [workspace.dependencies]
-everestrs = { git = "https://github.com/everest/everest-framework.git", rev = "01468678f5b627a96791f749f3139c3d169bf081" }
-everestrs-build = { git = "https://github.com/everest/everest-framework.git", rev = "01468678f5b627a96791f749f3139c3d169bf081" }
+everestrs = { git = "https://github.com/everest/everest-framework.git", rev = "cb6bf68d68d6c1fbe8ede1f1c5abe951a0507cc2" }
```
Comment (dorezyuk) [inline modules/Cargo.toml:11]: changes provided by https://github.com/EVerest/everest-framework/pull/168

File: modules/RsPaymentTerminal/src/logger.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+use env_logger::{Builder, Env};
```
Comment (dorezyuk) [inline modules/RsPaymentTerminal/src/logger.rs:1]: note: file is identical to the RsIskraMeter module and will be obsolete with https://github.com/Qwello/everest-framework/pull/31

File: types/money.yaml [hunk 3]
```diff
@@ -11,7 +11,6 @@ types:
     type: string
     minLength: 3
     maxLength: 3
-    example: EUR
```
Comment (dorezyuk) [inline types/money.yaml:14]: Not sure if this is part of the schema

File: modules/RsPaymentTerminal/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  terminal_id:
+    description: Payment terminal ID
+    default: "52525617"
+    type: string
+  currency:
+    description: Currency in ISO 4217 format
+    default: 978
+    type: integer
+  pre_authorization_amount:
+    description: pre_authorization_amount
```
Comment (dorezyuk) [inline modules/RsPaymentTerminal/manifest.yaml:20]: fixme

File: modules/RsPaymentTerminal/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  transactions_max_num:
+    description: >-
+      The maximum number of concurrent transactions. The payment terminal will
+      reject a transaction if it would exceed the maximum number of concurrent
+      transactions - in this case you need to finish a transaction to allow a
+      new one
+    default: 1
+    type: integer
+  password:
+    description: Password for the feig terminal.
```
Comment (barsnick) [inline modules/RsPaymentTerminal/manifest.yaml:42]: Capitalize "Feig" for consistency.

**Comment by SirVer** [comment]
this slipped by me, sorry for that. The rust code still looks good to me (after having seen the internal version) and the Bazel stuff is fine too.

**Comment by SirVer** [comment]
@hikinggrass @corneliusclaussen @caller This adds support for the CVEND Feig terminal and is the code we use on our 2nd gen. Could you please see if this is to your liking?

File: modules/RsPaymentTerminal/src/logger.rs (hunk 2 above)
Comment (hikinggrass) [reply modules/RsPaymentTerminal/src/logger.rs:1]: that should be obsolete now?

Comment (dorezyuk) [reply modules/RsPaymentTerminal/src/logger.rs:1]: I haven't merged the logging support into the everest/everest-framework yet

File: modules/RsPaymentTerminal/manifest.yaml [hunk 6]
```diff
@@ -0,0 +1,61 @@
+description: Payment terminal module for Feig devices.
```
Comment (Pietfried) [inline modules/RsPaymentTerminal/manifest.yaml:1]: nit: defaults for `ip`, `feig_serial` and `terminal_id` (and maybe `password`) do not look like reasonable. I'd recommend to use no default and therefore enforce proper configuration

**Review by Pietfried — APPROVED** [review]

File: modules/RsPaymentTerminal/manifest.yaml (hunk 6 above)
Comment (dorezyuk) [reply modules/RsPaymentTerminal/manifest.yaml:1]: true - was more like documentation feature -

File: modules/Cargo.lock [hunk 7]
```diff
@@ -770,53 +1810,173 @@ version = "0.52.4"
...
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "8f58c7ce1b7faa85a5c5a8b45b428c89cabdcd097bb472b7668037b291a8a20b"
+dependencies = [
+ "thiserror",
+]
+
+[[package]]
+name = "zvt"
+version = "0.1.1"
+source = "git+https://github.com/Qwello/zvt.git?rev=843ecf44ab592216bd4a483a07543c329d2baee1#843ecf44ab592216bd4a483a07543c329d2baee1"
```
Comment (hikinggrass) [inline modules/Cargo.lock:1918]: This (and the following packages) should now point to EVerest/zvt

**Pull request merged**